src/test/java/cs2103/v15_1j/jimjim/controller/JJControllerTest.java

```diff
@@ -123,6 +124,15 @@ public void testAdd() {
     public void testDelete() {
         controller.execute("Prepare for German exams");
         controller.execute("Buy flowers for her by Tuesday");
+        
+        for(DeadlineTask t: controller.getMasterList().getDeadlineTasksList()){
+        	controller.states.displayList.add(t);
+        }
+        
+        for(FloatingTask t: controller.getMasterList().getFloatingTasksList()){
+        	controller.states.displayList.add(t);
+        }
+        
```

Comment:
@SorataYuu Can it be done in controller instead of UI then? UI just pull the displayList from controller and display them.